File: solutions/fortran/darts/1/darts.f90
module darts
  implicit none

contains

  function score(x, y) result(points)
    real, intent(in):: x, y
    integer :: points
    if (x**2+y**2<=1) then
      points=10
    else if (x**2+y**2<=25) then
      points=5 
    else if (x**2+y**2<=100) then
      points=1
    else
      points=0
    end if
  end function score

end module darts

File: solutions/cpp/darts/2/darts.cpp
#include "darts.h"

namespace darts {
    int score(double x, double y){
        double sum_sq=x*x+y*y;
        if (sum_sq <= 1.0)
            return 10;
        else if (sum_sq <= 25.0)
            return 5;
        else if (sum_sq <= 100.0)
            return 1;
        else
            return 0;
    }
}
